client/my-sites/checklist/wpcom-checklist/index.jsx

```diff
@@ -110,7 +111,7 @@ export default connect( ( state, ownProps ) => {
 	const siteId = getSelectedSiteId( state );
 	const designType = getSiteOption( state, siteId, 'design_type' );
 	const isEligibleForChecklist = isEligibleForDotcomChecklist( state, siteId );
-	const isSection = isSiteSection( state );
+	const isSection = isSiteSection( state ) && ! isBlockEditorSectionInTest( state );
```

Comment:
What do you think about renaming this to something like `isEligibleSection` or something equally thrilling?

I think `isSection` no longer exactly describes the var.

Reply:
Good point. I hear that naming may be hard and/or important. :)